tests/autoyast/installation.pm

```diff
@@ -23,7 +23,7 @@ use utils;
 
 my $confirmed_licenses = 0;
 my $stage              = 'stage1';
-my $maxtime            = 2000;       #Max waiting time for stage 1
+my $maxtime            = 7200;       #Max waiting time for stage 1
```

Comment:
I recommend to not change this lightly. This is especially a problem since now you allow this step to take longer than the MAX_JOB_TIME which is 7200 by default so if the installation does not finish within that time you will end up with incomplete jobs without any logs which are very annoying and hard to review. I recommend to change that back. If you have a specific worker instance for local development where everything is slower use the backend variable SCALE_TIMEOUT in the worker configuration. If you found a scenario where the installation takes longer regardless of the worker then maybe it's a bug that the filesystem does not write that fast.

Reply:
Ok, I'm trying with the TIMEOUT_SCALE option (according to the code, it's this option and not TIMEOUT_SCALE :)).

Reply:
sounds better ;-)

Reply:
After multiple tests I had to set maxtime to 5000 to be able to install my VM, TIMEOUT_SCALE is good for assert_screen, wait_still_screen and other function but useless for the global timeout during autoyast installation...
For my openQA development I use ppc64le VM on a x86_64 server, so installation takes more time.

Another solution could be to try to support TIMEOUT_SCALE in autoyast/installation.pm, could it be a good idea?

Reply:
Yes, of course. I am sorry, I had an intermediate state of the test code in mind which we had to partially revert with

```
commit 49b05c17
Author: Rodion Iafarov <riafarov@suse.com>
Date:   Wed Jun 14 17:32:54 2017 +0200

    Introduce timeout for installation
    
    As part of commit 44e5ffe3 we have removed timeout in order to get
    better error handling if test fails to match any of possible needles.
    Whereas under some conditions, we may end up in infinite loop if
    assert_screen always matches a needle. So, we introduce timeout back,
    but try to keep advantages of previous implementation as well.
```

so we are back to the custom timeout implementation. Yes, in this case I really recommend to scale `$maxtime`, e.g. `my $maxtime = 2000 * get_var('TIMEOUT_SCALE', 1)` and then configure your worker accordingly. In the long run - as you are a SUSE employee as well - I recommend you can get a hold of a ppc64le machine (could be nested virt but not cross-architecture emulated) if you need to handle ppc64le more.

Reply:
Ok, I will change the maxtime value like you suggest, thanks! For the ppc64le, in qa-css we have bare metal server but more for SAP testing :) and sometimes I need to use my personal PC (I'm in working from home).

Reply:
There are more machines available on orthos.

> and sometimes I need to use my personal PC (I'm in working from home).

there is ssh for that ;-)

Reply:
I know for ssh, but it's in case of Internet or VPN access problem ;-)
Nevermind, the ```my $maxtime = 2000 * get_var('SCALE_TIMEOUT', 1)``` works well.

But maybe I need to create another PR for that? As it's not really ppc64le related.

Reply:
No need for a separate PR but please a single commit

Reply:
Sure!

Reply:
ugh, now I understand how I phrased it ambiguous. I meant *separate*, sorry about that. So one commit to scaling the maxtime parameter with SCALE_TIMEOUT, another one with your changes to ppc64le support. Can you split again?

Reply:
Ok, so 1 PR but 2 commits, no problem I will do this :)